.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -30,18 +33,92 @@ function getDeployTableMessage(platformResult) {
     }
 }
 
+/**
+ * Get PR's neccessary information
+ *
+ * @param {Number} [pr] PR number
+ * @returns {Promise<any>}
+ */
+function getPR(pr) {
```

Comment:
This will lead to a _lot_ of individual API requests to fetch each PR. Instead, here's what I would recommend (borrowed from your implementation in a recent PR). Declare this function in `GithubUtils`:

```javascript
/**
 * Fetch all pull requests given a list of PR numbers.
 *
 * @param {Array<Number>} pullRequestNumbers
 * @returns {Promise}
 */
static fetchAllPullRequests(pullRequestNumbers) {
    const oldestPR = _.first(_.sortBy(pullRequestNumbers));
    return this.octokit.paginate(this.octokit.pulls.list, {
        owner: GITHUB_OWNER,
        repo: EXPENSIFY_CASH_REPO,
        state: 'all',
        sort: 'created',
        direction: 'desc',
        per_page: 100,
    }, ({data}, done) => {
        if (_.find(data, pr => pr.number === oldestPR)) {
            done();
        }
        return data;
    })
        .then(prList => _.filter(prList, pr => _.contains(pullRequestNumbers, pr.number)));
}
```

And use that to get all the PRs from the list, and then just map the result to your custom `{mergedAt, hasCPStagingLabel}` shape